src/components/PrivacyPolicy/PspPrivacyInfo.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Link, Typography } from "@mui/material";
+import { Trans } from "react-i18next";
+
+const PrivacyLink = (props: {
+  href: string;
+  children?: string;
+  rel?: string;
+}) => {
+  const { href, children, rel = "noreferrer" } = props;
+  return (
+    <Link
+      href={href}
+      target="_blank"
+      rel={rel}
+      style={{ fontWeight: 600, textDecoration: "none" }}
+    >
+      {children}
+    </Link>
+  );
+};
+
+interface PspPrivacyInfoProps {
+  termsLink: string;
+  privacyLink: string;
+  pspName: string;
+}
+
+const PspPrivacyInfo = (props: PspPrivacyInfoProps) => {
+  const { termsLink, privacyLink, pspName } = props;
+
+  return (
+    <Typography variant="caption" component={"div"}>
+      <Trans
+        i18nKey="paymentCheckPage.disclaimer.psp"
+        values={{ pspName }}
+        components={{
+          privacy: <PrivacyLink href={privacyLink} />,
+          terms: <PrivacyLink href={termsLink} />,
+          psp: <div />,
```

Comment:
```suggestion
          psp: <span />,
```


Not using `span` (or an inline container) the PSP name will always be in a new line:
<img width="660" alt="Screenshot 2025-05-14 at 15 28 40" src="https://github.com/user-attachments/assets/477d03ef-dbac-4fd1-8ca6-88b70d982b96" />

Reply:
ok